src/utils/underscore.ts

```diff
@@ -0,0 +1,12 @@
+export default function underscore(camelCasedWord: string) {
```

Comment:
I wouldn't mind having lodash as a dependency as long as we make sure we only import the couple of functions we need from it.

Reply:
We seem to be [moving away from lodash](https://twist.com/a/1585/msg/756020/m/30577974), do we want to do this?